src/server/components/lists/searches/translators-interpreters.ts

```diff
@@ -38,21 +42,23 @@ export const translatorsInterpretersQuestionsSequence = [
 
 const serviceTypeToNoun: Record<string, string> = {
   translation: "translator",
-  interpretation: "interpreter"
+  interpretation: "interpreter",
 };
 
 function makeResultsTitle(country: string, servicesProvided: string[]): string {
-  const sanitisedServicesProvidedQuery = servicesProvided.map(service => serviceTypeToNoun[service]).filter(Boolean)
-  const interpretationOnly = sanitisedServicesProvidedQuery.includes(serviceTypeToNoun.interpretation) && sanitisedServicesProvidedQuery.length === 1;
+  const sanitisedServicesProvidedQuery = servicesProvided.map((service) => serviceTypeToNoun[service]).filter(Boolean);
+  const interpretationOnly =
+    sanitisedServicesProvidedQuery.includes(serviceTypeToNoun.interpretation) &&
+    sanitisedServicesProvidedQuery.length === 1;
   const article = interpretationOnly ? "an" : "a";
-  return `Find ${article} ${sanitisedServicesProvidedQuery.join(" or ")} in ${country}`
+  return `Find ${article} ${sanitisedServicesProvidedQuery.join(" or ")} in ${country}`;
 }
 
+function hasSworn(results: TranslatorInterpreterListItemGetObject[]): boolean {
+  return results.some((result) => result.jsonData.swornTranslator || result.jsonData.swornInterpreter);
```

Comment:
I couldn't see swornTranslator or swornInterpreter properties in TranslatorInterpreterFormWebhookData.  There's swornTranslations but nothing for interpreters.

Reply:
Good spot. I don't know why Typescript didn't get angry at me for that 🤔

Reply:
just another thought on this.  I believe these fields should be getting captured already.  As there's already a swornTranslations field being captured this makes me wonder if we need to differentiate between the two and therefore should we just be using jsonData.swornTranslations as the flag.  Same point in some ways as the one below so worth running it past Neil to verify this.